mod-dcb/src/main/resources/volaris/mod-dcb/features/cancelling-flow-for-pickup-flow.feature

```diff
@@ -0,0 +1,246 @@
+Feature: Testing Pickup Flow Cancellation
+
+  Background:
+    * url baseUrl
+
+    * callonce login testAdmin
+    * def okapitokenUser = okapitoken
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json, text/plain'  }
+    * configure headers = headersUser
+    * callonce variables
+
+  Scenario: Cancel DCB Transaction manually
```

Comment:
Please add one more scenario, try to cancel dcb transaction which is in closed status. This should give 4xx error

Reply:
fixed